src/components/encryption.ts

```diff
@@ -142,6 +147,8 @@ export class EncryptedEventBroker {
         const existingUserForRoom = this.userForRoom.get(event.room_id);
         if (existingUserForRoom) {
             log.debug(`${existingUserForRoom} is listening for ${event.event_id}`);
+            // XXX: Sometimes the sync stops working, calling this will wake it up.
+            await this.startSyncingUser(existingUserForRoom);
```

Comment:
This is unfortunate but sometimes even though a user is syncing for messages, they stop working. Calling this function *checks* that the user is syncing.